doc/manual/team.rst

```diff
@@ -54,6 +54,13 @@ become of importance when you run into problems.
    requests and replies is done through the web interface at
    |baseurlteam|.
 
+   Use the |problemseturlteam| page to get:
+
+     - all attachments such as 'testing tools' to problems,
+     - view the submission and acceptance ratio during the contest as an indicator of the difficulty of a problem,
```

Comment:
I like the brevity of this explanation, but I'm not sure if we might want to include a hint that more information is available when hovering the indicators.

Reply:
Feel free to propose an alternative text.